variables.go

```diff
@@ -0,0 +1,59 @@
+package main
```

Comment:
This should not be part of the main app, it should be with the helper file

Reply:
Done